Competition2017/src/competition/subsystems/agitator/AgitatorsManagerSubsystem.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.agitator;
+
+import org.apache.log4j.Logger;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import competition.subsystems.RobotSide;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.logging.RobotAssertionManager;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class AgitatorsManagerSubsystem extends BaseSubsystem {
```

Comment:
This shouldn't be plural. Also, if following the previous naming change, this should just be `AgitatorSubsystem`.